astropy/utils/iers/iers.py

```diff
@@ -705,15 +702,15 @@ def read(cls, file=None, readme=None, data_start=6):
         ``eopc04_IAU2000.62-now``), do something like this example with an
         excerpt that is used for testing::
 
+            >>> from astropy_iers_data import IERS_B_IAU2000_README
```

Comment:
Seeing this example, I think we should indeed add the readme to `astropy_iers_data` (it is a TODO right now), but it might also make sense to hide the package from the user here, i.e., do `from astropy.utils.iers import IERS_B, IERS_B_IAU2000_README`

Reply:
Agreed on both points.